File: ESTRUTURAS/1244/main.js
function strings() {
    return lines.shift().split(' ').sort((a, b) => b.length - a.length);
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let n = parseInt(lines.shift());
while (n--)
    console.log(strings().join(' '));

File: ESTRUTURAS/1244/main.dart
import 'dart:io';

List<String> strings() {
  return stdin.readLineSync()!.split(' ')..sort((a, b) => b.length - a.length);
}

void main(List<String> args) {
  int n = int.parse(stdin.readLineSync()!);
  while (n-- > 0) print(strings().join(' '));
}
